src/styles/blocks/_stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+  width: 80%;
+
+  &__star {
+    height: $star-size;
+    width: $star-size;
+    background-image: url("/images/star.svg");
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+  }
+
+  &--5 .stars__star,
```

Comment:
Why so many declarations? You just need one for 4 active stars

Reply:
@mbruhler  It was good that way. The amount of active-stars should be exactly the same as the value of the `stars--N `modifier